pydy/viz/static/js/dyviz/materials.js

```diff
@@ -14,7 +14,8 @@ DynamicsVisualizer.Materials = Object.extend(DynamicsVisualizer, {
 
         var self = this;
 
-        self.texture_base_path = "static/textures/"
+        // TODO : This `pydy-resources` directory should not be hard coded here!
+        self.texture_base_path = "pydy-resources/textures/"
```

Comment:
Sahil, how can we properly fix this? The textures need to be loaded relatively but this is a hard coded path.